.vscode/launch.json

```diff
@@ -0,0 +1,17 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+
+        {
+            "name": "Launch Package",
+            "type": "go",
+            "request": "launch",
+            "mode": "auto",
+            "program": "${fileDirname}",
+            "args": ["-driver=mysqldump", "-instance=e2e-spanner-testing", "-v=true", "-dump-file="]
```

Comment:
suggest reformatting this as to make it easier to edit

```
"args": [
    "-driver=mysqldump", 
    "-instance=e2e-spanner-testing", 
    "-v=true",
    "-dump-file="
]
```

Reply:
changed